bin/deploy

```diff
@@ -8,13 +8,13 @@ JEFFREY_TOKEN_VAR="JEFFREY_TOKEN_${DEPLOY_ENV}"
 BUCKET_VAR="BUCKET_${DEPLOY_ENV}"
 BUCKET_PREFIX_VAR="BUCKET_PREFIX_${DEPLOY_ENV}"
 AWS_KEY_VAR="AWS_KEY_${DEPLOY_ENV}"
-AWS_SECRET_VAR="AWS_KEY_${DEPLOY_ENV}"
+AWS_SECRET_VAR="AWS_SECRET_${DEPLOY_ENV}"
 
 export JEFFREY_TOKEN=${!JEFFREY_TOKEN_VAR}
 export BUCKET=${!BUCKET_VAR}
 export BUCKET_PREFIX=${!BUCKET_PREFIX_VAR}
-export AWS_KEY=${!AWS_KEY_VAR}
-export AWS_SECRET=${!AWS_SECRET_VAR}
+export AWS_ACCESS_KEY_ID=${!AWS_KEY_VAR}
```

Comment:
Pretty much. I thought I had set them correctly in the previous pr where it worked on my local but I had not. I'm not sure how but I ended up with weird file conflicts locally that I believe masked the issue.